media-sound/beets/beets-1.4.9-r3.ebuild

```diff
@@ -97,76 +120,44 @@ distutils_enable_tests pytest
 python_prepare_all() {
 	distutils-r1_python_prepare_all
 
-	rm_use_plugins() {
```

Comment:
Is it correct that all of this gets dropped?

Reply:
Ah yes check the previous feedback on this,
but it was done to simplify the ebuild.

Reply:
OK, clear, thanks.

Reply:
Thank you Simon for taking the time of the review !